message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -870,6 +870,9 @@
       <entry value="512" name="EKF_PRED_POS_HORIZ_ABS">
         <description>Set if EKF's predicted horizontal position (absolute) estimate is good.</description>
       </entry>
+      <entry value="1024" name="EKF_INITALIZED">
```

Comment:
Speeling.

Reply:
fxide, should probability turn on the spell check for xml's....